drivers/storage/portworx/deployment.go

```diff
@@ -771,8 +771,13 @@ func (t *template) getCloudProvider() string {
 }
 
 func (t *template) getArguments() []string {
+	clusterID := t.cluster.Name
+	if t.cluster.Annotations != nil && t.cluster.Annotations[pxutil.AnnotationClusterID] != "" {
```

Comment:
nit: In golang, you don't need to check if a map is nil. You can directly try to check the key.
```
if t.cluster.Annotations[pxutil.AnnotationClusterID] != "" {
```